redirect.py

```diff
@@ -41,8 +41,20 @@ def get(self):
           self.response.write(json.dumps(result))
       elif self.request.get('site') == 'comments':
           urlfetch.set_default_fetch_deadline(10)
+          startIndex = self.request.get('start')
+
+          try:
+            startIndex = int(startIndex)
+          except:
+            startIndex = 0
+
+          maxResults = self.request.get('max')
+          try:
+            maxResults = int(maxResults)
+          except:
+            maxResults = 0
           self.response.headers.add_header("Access-Control-Allow-Origin", "*")
-          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId')).execute()
+          result = monorail.issues().comments().list(projectId='chromium', issueId=self.request.get('issueId'), maxResults=max, startIndex=startIndex).execute()
```

Comment:
how about we forego the maxResults in that case and just ask people to page. That is what we do with issues right now as is?